packages/site/src/pages/index.tsx

```diff
@@ -304,9 +301,8 @@ const Index = () => {
       }
     }
 
-    const currentChainId = (await window.ethereum.request({
-      method: 'eth_chainId',
-    })) as string;
+    const currentChainId = window.ethereum.chainId as string;
```

Comment:
this is deprecated/removed in newer versions https://github.com/MetaMask/providers/blob/main/CHANGELOG.md#removed
let's utilize eth_chainId like before

Reply:
I have to add it intentionally, as that one returning only the mainnet chain id not the connected one!